mtags/src/main/scala-3/scala/meta/internal/mtags/MtagsEnrichments.scala

```diff
@@ -121,4 +130,25 @@ object MtagsEnrichments extends CommonMtagsEnrichments:
     def backticked: String =
       KeywordWrapper.Scala3.backtickWrap(s)
 
+  extension (search: SymbolSearch)
+    def symbolDocumentation(symbol: Symbol)(using
+        Context
+    ): Option[SymbolDocumentation] =
```

Comment:
(Don't need to work in this PR)
What do you think about renaming `SymbolDocumentation` to `MetalsSymbolInformation` or something?
`SymbolDocumentation` has more than documentation infomration such as `parameters`, `defaultValue`, and `typeParameters`. 

At the first glance, I was confused like "why do we need `SymbolDocumentation` for getting default values of parameters ... 🤔 ".

Reply:
Though it's mostly about the documentation :thinking: